flask_monitoringdashboard/templates/endpoint/outliers.html

```diff
@@ -1,5 +1,6 @@
 {% extends "endpoint/main.html" %}
 {% block graph_content %}
+    <p> <strong>Mean CPU Percent: </strong>{{ mean }}</p>
```

Comment:
"Percentages" maybe?

Reply:
Yeah, that's better. I will include it in the next commit.